src/SarifCli/SarifValidationContext.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    public class SarifValidationContext : IAnalysisContext
+    {
+        public bool IsValidAnalysisTarget
+        {
+            get
+            {
+                return Path.GetExtension(TargetUri.LocalPath).Equals(".sarif", StringComparison.OrdinalIgnoreCase);
+            }
+        }
+
+        public IAnalysisLogger Logger { get; set; }
+
+        public string MimeType
+        {
+            get { return "text/x-sarif"; }
```

Comment:
>"text/x-sarif" [](start = 25, length = 14)

DRY this out by referencing a constant elsewhere (such as used by mime detection utility)